repositories/UserRepository.py

```diff
@@ -1,15 +1,36 @@
+import string
+
+from libsql_client import ResultSet
+
+from dtos import UserDto
 from repositories.BaseRepository import BaseRepository
 
 
 class UserRepository(BaseRepository):
     INSERT_USER_STATEMENT = "INSERT INTO user (username, password_hash, password_salt, email, name) VALUES (?, ?, ?, ?, ?)"
+    SELECT_USER_BY_USERNAME = "SELECT * FROM user where user.username = ?"
+    DELETE_BY_USERNAME = "DELETE FROM user WHERE user.username = ?"
 
-    def __init__(self, isProd):
-        super().__init__(isProd)
 
-    def registerUser(self, userDto):
+    def __init__(self):
+        super().__init__()
+
+    def registerUser(self, user: UserDto):
         self.execute(
             UserRepository.INSERT_USER_STATEMENT,
-            [userDto.username, userDto.password,
-             userDto.passwordSalt, userDto.email,
-             userDto.name])
+            [user["username"], user["password"],
+             user["passwordSalt"], user["email"],
+             user["name"]])
```

Comment:
If the `__getattr__()` function is committed:
```suggestion
            [user.username, user.password,
             user.passwordSalt, user.email,
             user.name])
```